src/components/course/EnrollButton.jsx

```diff
@@ -33,20 +35,20 @@ export default function EnrollButton() {
     pacingType,
     courseUuid,
   } = activeCourseRun;
-  const [isEnrollDisabled, setIsEnrollDisabled] = useState(false);
-  const [emailOptIn] = useState(false);
-  const [enrollmentSubmitted, setEnrollmentSubmitted] = useState(false);
 
-  /*
-   * TODO: this behavior mimics the existing B2C enrollment flow. we will want
-   * to redirect learners to the basket flow instead of the track selection page
-   */
-  useEffect(() => {
-    if (enrollmentSubmitted) {
-      // redirect to track selection page
-      window.location.href = `${process.env.LMS_BASE_URL}/course_modes/choose/${key}`;
-    }
-  }, [enrollmentSubmitted]);
+  // This enrollment URL assumes that the learner has access to the course through their subscription license, and they
+  // are using the license to enroll.
+  // TODO: Update to conditionally use the DSC flow with a license uuid only when the learner is actually using a
+  // license.
+  const enrollOptions = {
+    license_uuid: subscriptionLicense.uuid,
+    course_id: key,
```

Comment:
Just to confirm / double check: the course `key` is what we want here, not the actively selected course run `key`, correct?

Reply:
Ah I think it is the course run key

Reply:
I think this is using the active course run key though?

Reply:
Ah, yup. You're right, it is the active course run key :) Just wanted to double check!